src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -380,16 +427,15 @@ public void onTestFailure(ITestResult result) {
         log("onException",
             "driver has no ability to catch screenshot or html source - driver may died", false);
       }
-
       String exception = escapeHtml(result.getThrowable().toString() + "\n"
           + ExceptionUtils.getStackTrace(result.getThrowable()));
+      //TODO: Check imageCounter
 
-      StringBuilder builder = new StringBuilder();
-      builder.append("<tr class=\"error\"><td>error</td><td><pre>" + exception
-          + "</pre></td><td> <br/><a href='screenshots/screenshot" + imageCounter
-          + ".png'>Screenshot</a><br/><a href='screenshots/screenshot" + imageCounter
-          + ".html'>HTML Source</a></td></tr>");
-      CommonUtils.appendTextToFile(logPath, builder.toString());
+      List<String> classList = new ArrayList<>();
+      classList.add(ERROR_CLASS);
+//      classList.add(STACKTRACE_CLASS);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)